config/initializers/paperclip.rb

```diff
@@ -1,6 +1,6 @@
 Paperclip::Attachment.default_options.merge!(
   path: ":rails_root/uploads/:class/:attachment/:id_partition/:style/:filename",
-  url: "/uploads/:class/:id/:style"
+  url: "/:locale/m/:mission/:class/:id/:style"
```

Comment:
`media` is interpolated by `:class` in this case, it gives the fully qualified class name separated by slashes, so images will be in `media/images`, while audio is in `media/audio`, etc. The route expects `media` explicitly so that I don't need to parse the subclass from the params.